### PR TITLE
Build a personal portfolio site: Adding the toolbar and headline sections

### PR DESCRIPTION
**Pull request by selma-belhadj**
In this PR I have completed the following tasks:

- Setting up the HTML and CSS Linters correctly
- Building the toolbar (or header) and the headline sections  of the portfolio
- Using Flexbox in all 2 sections.
- Using images and backgrounds to enhance the look of the website.
- Making the README file should be descriptive in a professional way.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>    
+    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@100;200;300;400;500;600;700;800;900&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <h2><a href="#">Welcome</a></h2>
+        <i class="fa fa-bars" aria-hidden="true"></i>
```
Comment (leolpaz) [inline index.html:16]: - [ ] Great work with getting the icon, but we could be a little more semantic here, in line with HTML and CSS best practices, kindly wrap your icon inside a `<nav></nav>` tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    </header>
+    <section class="home-page">
+        <h1 class="intro">Hey there. I am Selma <br><span>I am a software<br> developer</span></h1>
```
Comment (leolpaz) [inline index.html:19]: - [ ] To be more in line with semantic HTML here, it'd be better if "I am a software developer" was instead on an h2 tag below, it's ok for both of them to have the same size, you can change by adding a class of "sub-intro", the reason is that the h1 tag has implications in the accessibility of the page for screen readers, and "I am a software developer" is better treated as a subtitle

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,72 @@
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Inter', sans-serif;
+}
+
+body {
```
Comment (leolpaz) [inline styles.css:9]: - [ ] On this project we have to follow the Figma design, you can see on the screenshots below, that on the template the background "bleeds" through the padding, kindly change it so it better follows the design, changing the padding from the body to the header and the section tag class, helps achieve that effect, feel free to tweak the background-size from cover if you feel the need to
Figma:
image
Yours: 
image

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+  font-weight: 600;
+}
+
+header .fa-git {
+  font-weight: 500;
+  font-size: 20px;
+  cursor: pointer;
+}
+
+.home-page {
```
Comment (leolpaz) [inline styles.css:33]: - [ ] Great job positioning the sub-contents of this section, but it'd be better if you used Flexbox on the container to position the items, I understand they already look great, but one of the goals of this project is to greatly improve your proficiency with Flexbox, so kindly use Flex on this section to help position the sub-items and adjust the margins accordingly😃
image

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @selma-belhadj , 

You've done an amazing job so far, I'd like to highlight the following:
- Great README
- PR title and summary are descriptive
- Great understanding of HTML

Some issues need to be resolved before your approval, but you're almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by selma-belhadj** [comment]
Thank you for your response. I have made the required changes so what's next?

**Review by MrBrN197 — APPROVED** [review]
## Hi, @selma-belhadj  👏

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Approved ✔

## To Highlight 👏
- ✔ Your design matches the Figma template really well.
- ✔ You have a professional README

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+    <link rel="stylesheet" href="styles.css">
+    <title>Portfolio</title>
+</head>
+<body>
+    <header>
+        <h2 class="welcome"><a href="#">Welcome</a></h2>
+        <nav><i class="fa fa-bars" aria-hidden="true"></i></nav>
+    </header>
+    <section class="home-page">
+        <h2 class="intro">Hey there. I'm Selma <br><span>I am a software<br> developer</span></h2>
```
[the remark this answers is not in the document]
Comment (selma-belhadj) [reply index.html]: Thank you for your review. I've tried to put the p tag but I had a linter error:
Screenshot 2021-11-26 at 12 27 37
So, I'll keep it as a span tag

File: package.json [hunk 6]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "my-1st-portfolio",
+  "version": "1.0.0",
+  "description": "![](https://img.shields.io/badge/Microverse-blueviolet)",
```
[the remark this answers is not in the document]
Comment (selma-belhadj) [reply package.json]: All right done @MrBrN197 thanks a lot

**Pull request merged**